client/src/components/content/ModelerSidebar/ModelerSidebar.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from './ModelerSidebar.module.css';
+import PropertiesPanel from '../PropertiesPanel/PropertiesPanel';
+import { Layout, Button } from 'antd';
+
+const { Sider } = Layout;
+
+const ModelerSidebar = (props) => {
+  return (
+    <Sider className={styles.sider}>
+      {props.modeler && <PropertiesPanel modeler={props.modeler} />}
+      <Button
+        type='primary'
+        style={{
+          width: '80%',
+          marginTop: '10px',
+          marginLeft: '30px',
+          marginRight: '30px',
+        }}
+        onClick={props.getBpmnDiagramRobot}
```

Comment:
This method downloads the .robot file, right? Should we rather change the name to `getRobotFile?

Reply:
Thats a good idea. I will do that.